CHANGELOG.md

```diff
@@ -1,6 +1,11 @@
 
 # Changelog
 
+## 0.5.0
+* Support signing with an AWS KMS private key via the `key_kms` feature. (thank you @puiterwijk)
+* Abstract Openssl operations (thank you @raoulstrackx)
+* Update and declare MSRV to 1.58
```

Comment:
nit: In addition to the MSRV, would it be beneficial to mention new dependencies?  (aws-config, tokio)

Reply:
They only show up when activating `key_kms` and I don't see a good reason to list them specifically. They'll show up on crates.io and are included in Cargo.toml.